microsoftteams_connector.py

```diff
@@ -1113,6 +1115,148 @@ def _handle_create_meeting(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS, status_message="Meeting Created Successfully")
 
+    def _handle_get_channel_message(self, param):
+        """This function is used to get message from specified channel in a Microsoft Teams group.
+
+        :param param: Dictionary of input parameters
```

Comment:
Adding typing to the header would be cleaner and more readable

Reply:
Changes done.